addons/sms/static/tests/qunit_suite_tests/components/sms_button_tests.js

```diff
@@ -17,20 +19,27 @@ QUnit.module(
                             message: { string: "message", type: "text" },
                             foo: { string: "Foo", type: "char", default: "My little Foo Value" },
                             mobile: { string: "mobile", type: "text" },
+                            start: { string: "Start Date", type: "datetime" },
+                            stop: { string: "Stop Date", type: "datetime" },
                         },
                         records: [
                             {
                                 id: 1,
                                 message: "",
                                 foo: "yop",
                                 mobile: "+32494444444",
+                                start: moment().format("YYYY-MM-DD HH:00:00"),
+                                stop: moment().add(4, "hours").format("YYYY-MM-DD HH:00:00"),
```

Comment:
Why not defining a static date instead of using moment to have the current date?

Reply:
> Why not defining a static date instead of using moment to have the current date?

If using the static date we have to add `patchDate` function so I avoided it. But now used.thanks